### DISCUSSION TITLE
PKCS12 key store mac invalid - wrong password or corrupted file.

### DISCUSSION THREAD

**Discussion by tonnoone** [malikzh/NCANode#204] [Q&A]
Подскажите, что это за ошибка и как её лечить?
java.io.IOException: PKCS12 key store mac invalid - wrong password or corrupted file.

**Comment by kmara94** [comment]
Попробуйте изменить права доступа к папке где хранится ЭЦП. Если на линухе: sudo chown -R user:user <folder>
